provision/kubernetes/deploy.go

```diff
@@ -507,8 +499,18 @@ func createAppDeployment(ctx context.Context, client *ClusterClient, depName str
 		return false, nil, nil, errors.WithStack(err)
 	}
 	var hcData hcResult
-	if process == webProcessName && len(processPorts) > 0 {
-		//TODO: add support to multiple HCs
+	// NOTE: Here is the code that create probes for HEALTHCHECK!
+	if len(yamlData.Processes) > 0 {
+		var healthcheck *provTypes.TsuruYamlHealthcheck
+		healthcheck, err = yamlData.GetHCFromProcessName(process)
```

Comment:
If user defines a usual global healthcheck block outside the new process structure it could be used instead returning an error ? Or can we send some kind of warning informing it will be ignored ?

Reply:
We could return a warning stating that global healthchecks and `Procfile` are going to be ignored when using the new process structure.